zigbee-console-common/src/main/java/org/bubblecloud/zigbee/ZigBeeConsole.java

```diff
@@ -387,6 +395,54 @@ public boolean process(final ZigBeeApi zigbeeApi, final String[] args) {
     }
 
     /**
+     * Prints list of devices to console.
+     */
+    private class NodesCommand implements ConsoleCommand {
+        /**
+         * {@inheritDoc}
+         */
+        public String getDescription() {
+            return "Lists node information.";
+        }
+        /**
+         * {@inheritDoc}
+         */
+        public String getSyntax() {
+            return "nodes";
+        }
+        /**
+         * {@inheritDoc}
+         */
+        public boolean process(final ZigBeeApi zigbeeApi, final String[] args) {
+            final List<ZigBeeNode> nodes = zigbeeApi.getNodes();
+            for (int i = 0; i < nodes.size(); i++) {
+                final ZigBeeNode node = nodes.get(i);
+                print("IEEE Address     : " + node.getIeeeAddress());
+                print("Network Address  : " + String.format("%04X", node.getNetworkAddress()));
```

Comment:
This displays in hexadecimal but everything else is in decimal.  I'd certainly prefer hex, but because everything else is decimal it should be the same I think.